### PR TITLE
[mimir-distributed] add named port memberlist to pod container

### PR DESCRIPTION
**Pull request by ortuman**
Signed-off-by: Miguel Ángel Ortuño <ortuman@gmail.com>

Add memberlist container named port to all components that depend on it.

### CODE REVIEW

File: charts/mimir-distributed/templates/alertmanager/alertmanager-dep.yaml [hunk 1]
```diff
@@ -80,6 +80,9 @@ spec:
             - name: grpc
               containerPort: {{ include "mimir.serverGrpcListenPort" . }}
               protocol: TCP
+            - name: memberlist
```
Comment (krajorama) [inline charts/mimir-distributed/templates/alertmanager/alertmanager-dep.yaml:83]: alertmanager has a statefulset deployment option also!

**Review by krajorama — CHANGES_REQUESTED** [review]
Need to add the new named template to templates/_helpers.tpl.
If we chose to hardcode the number, we should add a validation against changing via mimir.config. Otherwise should have a default hardcoded value and try to grab the bind port via mimir.calculatedConfig (see other examples). I think we can go with the second option as it's not referenced anywhere in mimir.config, so no infinite loop....

File: charts/mimir-distributed/templates/alertmanager/alertmanager-dep.yaml (hunk 1 above)
Comment (ortuman) [reply charts/mimir-distributed/templates/alertmanager/alertmanager-dep.yaml:83]: done @ b9d9200db8a18a37fc6340fa95a512b0371cd9db

**Review by krajorama — APPROVED** [review]
LGTM

**Pull request merged**